scalyr_agent/copying_manager.py

```diff
@@ -289,23 +289,24 @@ def __path_in_globs( self, path, path_globs ):
 
         return None
 
-    def add_log_config( self, monitor, log_config ):
+    def add_log_config( self, monitor_name, log_config ):
```

Comment:
Was there a reason to change this to `monitor_name`? I'm guessing this might be the reason unit tests are failing. Smoketests pass though which may suggest it safe to keep this but alter failing unit test